src/TLuaInterpreter.cpp

```diff
@@ -15835,3 +15836,106 @@ int TLuaInterpreter::getMouseEvents(lua_State * L)
+        }
+        if (key == "mapperShowMapInfo") {
+            host.mMapInfoContributors.insert(getVerifiedString(L, __func__, 2, "value"));
+            host.mpMap->mpMapper->slot_updateInfoContributors();
+            return success();
+        }
+        if (key == "mapperHideMapInfo") {
+            host.mMapInfoContributors.remove(getVerifiedString(L, __func__, 2, "value"));
+            host.mpMap->mpMapper->slot_updateInfoContributors();
+            return success();
+        }
+        if (key == "mapperShow3dView") {
+            host.mpMap->mpMapper->slot_toggle3DView(getVerifiedBool(L, __func__, 2, "value"));
+            return success();
+        }
+        if (key == "mapperPanelVisible") {
+            host.mpMap->mpMapper->slot_setMapperPanelVisible(getVerifiedBool(L, __func__, 2, "value"));
+            return success();
+        }
+        if (key == "mapperShowRoomBorders") {
+            host.mMapperShowRoomBorders = (getVerifiedBool(L, __func__, 2, "value"));
+            return success();
+        }
+        if (key == "generalEnableGMCP") {
+            host.mEnableGMCP = getVerifiedBool(L, __func__, 2, "value");
+            return success();
+        }
+        if (key == "generalEnableMSDP") {
+            host.mEnableMSDP = getVerifiedBool(L, __func__, 2, "value");
+            return success();
+        }
+        if (key == "generalEnableMSSP") {
+            host.mEnableMSSP = getVerifiedBool(L, __func__, 2, "value");
+            return success();
+        }
+        if (key == "generalEnableMSP") {
+            host.mEnableMSP = getVerifiedBool(L, __func__, 2, "value");
+            return success();
+        }
+        if (key == "inputLineStrictUnixEndings") {
+            host.mUSE_UNIX_EOL = getVerifiedBool(L, __func__, 2, "value");
+            return success();
+        }
+        if (key == "inputLineCommandSeparator") {
+            host.mCommandSeparator = getVerifiedString(L, __func__, 2, "value");
+            return success();
+        }
+        if (key == "mainDisplayFixUnnecessaryLinebreaks") {
+            host.set_USE_IRE_DRIVER_BUGFIX(getVerifiedBool(L, __func__, 2, "value"));
+            return success();
+        }
+        if (key == "specialForceCompressionOff") {
+            host.mFORCE_NO_COMPRESSION = getVerifiedBool(L, __func__, 2, "value");
+            return success();
+        }
+        if (key == "specialForceGAOff") {
+            host.mFORCE_GA_OFF = getVerifiedBool(L, __func__, 2, "value");
+            return success();
+        }
+        if (key == "specialForceCharsetNegotiationOff") {
+            host.mFORCE_CHARSET_NEGOTIATION_OFF = getVerifiedBool(L, __func__, 2, "value");
+            return success();
+        }
+        if (key == "specialForceMxpNegotiationOff") {
+            host.mFORCE_MXP_NEGOTIATION_OFF = getVerifiedBool(L, __func__, 2, "value");
+            return success();
+        }
+    }
+
+    lua_pushboolean(L, false);
```

Comment:
Also need to mention in the Debug console what was changed, so scripts aren't sneakily changing things from under you - example https://github.com/Mudlet/Mudlet/blob/development/src/TLuaInterpreter.cpp#L13258-L13260